dcos_launch/config.py

```diff
@@ -242,6 +242,18 @@ def get_validated_config(user_config: dict, config_dir: str) -> dict:
     'config_dir': {
         'type': 'string',
         'required': False},
+    'dcos_config': {
+        'type': 'dict',
+        'required': True,
+        'allow_unknown': True,
+        'default_setter': lambda doc: yaml.load(util.read_file(os.path.join(doc['genconf_dir'], 'config.yaml'))),
+        'schema': {
+            'calico_network_cidr': {
+                'type': 'string',
+                'required': True,
```

Comment:
how about we add a default here to 172.28.0.0/16 ?

Reply:
I advocated for this as the default as well at https://github.com/dcos-terraform/terraform-aws-dcos/pull/77#discussion_r376814907, but there was a desire to keep it mandatory and not have a default. This is just keeping parity with that.